app-text/a2ps/a2ps-4.15.6.recipe

```diff
@@ -75,6 +75,9 @@ defineDebugInfoPackage a2ps$secondaryArchSuffix \
 
 BUILD()
 {
+	# workaround for missing paperspecs config file in haikuporter chroot
+	echo A4,210,297,mm > $(finddir B_SYSTEM_SETTINGS_DIRECTORY)/paperspecs
+
```

Comment:
In psutils, I added this because it didn't build otherwise. If this built successfully before, it doesn't need a paperspecs configuration at build time, so this can be left out.